src/main/java/com/yelp/nrtsearch/server/luceneserver/NRTPrimaryNode.java

```diff
@@ -159,6 +162,10 @@ Iterator<TransferStatus> getTransferStatusIteratorForClient(ReplicationServerCli
     }
   }
 
+  private double getCurrentMaxMergePreCopyDurationSec() {
+    return indexStateManager.getLiveSettings().getMaxMergePreCopyDurationSec().getValue();
```

Comment:
This would be something like `indexStateManager.getCurrent().getMaxMergePreCopyDurationSec()`